src/components/Accordion/Accordion.tsx

```diff
@@ -36,6 +38,19 @@ const AriaAccordionItem: FC<AriaAccordionItemProps> = ({ item, state, header })
     const { buttonProps, regionProps } = useAccordionItem({ item }, state, triggerRef);
     const isOpen = state.expandedKeys.has(item.key) && item.props.children;
     const { isFocusVisible, focusProps } = useFocusRing();
+    const [isActive, setIsActive] = useState(false);
+    const isActiveByDefault = header.active;
+
+    useEffect(() => {
+        if (isActiveByDefault) {
+            state.toggleKey(item.key);
+        }
+        // We add a timeout to avoid isActive being set to true before
+        // first re-render and play animations on load.
+        setTimeout(() => setIsActive(true), 50);
```

Comment:
```suggestion
        // We add a timeout to avoid isActive being set to false before
        // the first re-render, so that the animations won't play on load
        setTimeout(() => setIsActive(false), 50);
```